tools/modulelint.py

```diff
@@ -45,8 +45,28 @@ def setUp(self):
     def testDockerFromBaseruntime(self):
         self.assertTrue(self.dp.check_baseruntime())
 
-    def testDockerRunMicrodnf(self):
-        self.assertTrue(self.dp.check_microdnf())
+    def testDockerNodocs(self):
+        self.start()
+        # This returns a list of packages defined in config.yaml for testing
+        # e.g. ["bash", "rpm", "memcached"] in case of memcached
+        pkgs = self.backend.getPackageList()
+        for pkg in pkgs:
+            docs = self.run("rpm -qad %s" % pkg).stdout
+            for doc in docs.strip().split('\n'):
+                self.assertNotEqual(0, self.run("test -f %s" % doc, ignore_status=True).exit_status)
```

Comment:
@ignatenkobrain self.run runs test inside the container. The tests are called outside of container. https://github.com/fedora-modularity/meta-test-family/blob/master/moduleframework/module_framework.py#L415